### PR TITLE
Added dedicated Australian server

### PR DESCRIPTION
**Pull request by Stuntaneous**
auslanplay.ddns.net:11451

### CODE REVIEW

**Comment by GreatWizard** [comment]
Oh OVH australia :)
Thanks to host this server!

**Comment by GreatWizard** [comment]
Can you move you server in between the rust and dotnet one ? I try to keep the list ordered by type of servers

File: public/data/servers.json [hunk 1]
```diff
@@ -216,4 +216,11 @@
     "platform": "switch",
     "type": "dotnet"
   }
```
Comment (GreatWizard) [inline public/data/servers.json:218]: You need to add a "coma" here ;)

**Comment by Stuntaneous** [comment]
Hope that does the trick.

**Comment by Stuntaneous** [comment]
Btw, unrelated: bluehouse.servepics.com is blocked for phishing by Malwarebytes.

**Pull request merged**